attestation-service/docs/parsed_claims.md

```diff
@@ -64,14 +64,27 @@ The following fields always exist.
 - `tdx.td_attributes.perfmon`: A boolean value that indicates whether the TD is allowed to use Perfmon and PERF_METRICS capabilities.
 - `tdx.td_attributes.protection_keys`: A boolean value that indicates whether the TD is allowed to use Supervisor Protection Keys.
-- `tdx.td_attributes.septve_disable`:  A boolean value that determines whether to disable EPT violation conversion to #VE on TD access of PENDING pages.
-- `tdx.advisory_ids`: List of Intel® Product Security Center Advisories
-- `tdx.collateral_expiration_status`:  Expected 0, if none of the inputted collateral has expired as compared to the inputted expiration_check_date.
+- `tdx.td_attributes.septve_disable`: A boolean value that determines whether to disable EPT violation conversion to #VE on TD access of PENDING pages.
+- `tdx.advisory_ids`: List of Intel® Product Security Center Advisories.
+- `tdx.collateral_expiration_status`: Expected 0, if none of the inputted collateral has expired as compared to the inputted expiration_check_date.
+- `tdx.earliest_expiration_date`: Date time value in RFC3339 format - The earliest nextUpdate value, or expiration date, among all collaterals.
+- `tdx.earliest_issue_date`: Date time value in RFC3339 format - The earliest issueDate among all collaterals.
+- `tdx.is_cached_keys`: A boolean value that indicates whether platform root keys are cached by SGX Registration Backend. _Note: this field is only provided if sgx_type is set to either "scalable" or "Scalable with Integrity"._
+- `tdx.is_dynamic_platform`: A boolean value that indicates whether a platform can be extended with additional packages. _Note: this field is only provided if sgx_type is set to either "scalable" or "Scalable with Integrity"._
+- `tdx.is_smt_enabled`: A boolean value that indicates whether a platform has SMT (simultaneous multithreading) enabled. _Note: this field is only provided if sgx_type is set to either "scalable" or "Scalable with Integrity"._
+- `tdx.latest_issue_date`: Date time value in RFC3339 format - The latest issueDate value among all collaterals.
+- `tdx.pck_crl_num`: Indication of the freshness of the PCK cert used.
+- `tdx.platform_provider_id`: The Platform Provisioning ID.
+- `tdx.root_ca_crl_num`: Indication of the freshness of the Root CA cert used.
+- `tdx.root_key_id`: ID of the collateral’s root signer (hash of Root CA’s public key SHA-384).
+- `tdx.sgx_type`: The type of memory used in SGX. Can be one of (`Standard`, `Scalable`, `Scalable with Integrity`).
+- `tdx.tcb_date`: Date time value in RFC3339 format - Earliest date between tcbInfo and qeIdentity.
+- `tdx.tcb_eval_num`: Indication of the freshness of the reference values used.
+- `tdx.tcb_status`: TCB status which can be one of: OK, Min, OutOfDate, OutOfDateConfigurationNeeded, InvalidSignature, Revoked, Unspecified, SoftwareHardeningNeeded, ConfigurationAndSoftwareHardeningNeeded, TdRelaunchAdvised, TdRelaunchAdvisedConfigurationNeeded, Max
```

Comment:
```suggestion
- `tdx.tcb_status`: TCB status which can be one of: `OK`, `Min`, `OutOfDate`, `OutOfDateConfigurationNeeded`, `InvalidSignature`, `Revoked`, `Unspecified`, `SoftwareHardeningNeeded`, `ConfigurationAndSoftwareHardeningNeeded`, `TdRelaunchAdvised`, `TdRelaunchAdvisedConfigurationNeeded`, `Max`
```

Reply:
Applied

Reply:
tcb_status could also stay unchanged in this changeset